examples/pdc_resource_tests.rb

```diff
@@ -172,6 +172,19 @@ def token
   end
 end
 
+describe PDC::V1::ContentDeliveryRepo do
+  let(:repo_class) { PDC::V1::ContentDeliveryRepo }
+  let(:existing_repo) { repo_class.all!.first }
+
+  describe '#find' do
```

Comment:
Thanks sthaha. Added. Always instinctively think the file with suffix 'test' is a test file :)